javascript-bank/account.js

```diff
@@ -0,0 +1,45 @@
+/* exported Account */
+class Account {
+  constructor(number, holder) {
+    this.number = number;
+    this.holder = holder;
+    this.transactions = [];
+    this.deposit = this.deposit.bind(this);
+    this.withdraw = this.withdraw.bind(this);
+    this.getBalance = this.getBalance.bind(this);
```

Comment:
You shouldn't need to bind these, since we're not passing them as callbacks anywhere.